.github/workflows/tests.yml

```
@@ -1,23 +1,38 @@
-      
-      - name: Install dependencies
-        run: composer install --profile --ignore-platform-reqs 
-        
-      - name: Run Tests
-        run: php -S localhost:8000 tests/router.php &
-          composer test
 
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          install: true
```

actionlint (1.7.10)

avoid using deprecated input "install" in action "docker/setup-buildx-action@v3": "docker buildx install" command is deprecated and will be removed in a future release, use BUILDX_BUILDER environment variable instead

(action)